modules/rust-wasm-test/Cargo.toml

```diff
@@ -12,7 +12,6 @@ bench = false
 
 [dependencies]
 spacetimedb = { path = "../../crates/bindings" }
-spacetimedb-lib = { path = "../../crates/lib" } # TODO: this should not ever be necessary. `spacetimedb` should provide all module dependencies.
```

Comment:
This was just cause I used crate to test, and I noticed that depending on lib directly like this enabled the default features and brought the dependency count way back up.